dotcom-rendering/src/web/lib/renderContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import { ContainerLayout } from '../components/ContainerLayout';
+import { DynamicFast } from '../components/DynamicFast';
+import { FixedLargeSlowXIV } from '../components/FixedLargeSlowXIV';
+
+type Props = {
+	trails: DCRFrontCard[];
+	containerType: DCRContainerType;
+};
+
+const renderContainer = ({ trails, containerType }: Props) => {
+	switch (containerType) {
+		case 'dynamic/fast':
+			return <DynamicFast trails={trails} />;
+		case 'fixed/large/slow-XIV':
+			return <FixedLargeSlowXIV trails={trails} />;
+		default:
+			// TODO: This default allows us to render fronts in-development where we might
+			// not support all the container types, but it should be removed / re-investigated
+			// before fronts are released
+			return <DynamicFast trails={trails} />;
+	}
+};
+
+export const renderFrontContainer = ({
```

Comment:
Is it worth maybe bringing this logic back out into `FrontLayout` and only having the decide container logic here? Our layout files _tend_ to be where these `ContainerLayout` components live

Reply:
👍  At first I was concerned that the logic for deciding a `trails` might grow more complex and therefor having it abstracted would be useful, but I think since it's looking relatively simple, keeping it in `FrontLayout` should be fine for now!